Application/Education/Certificate/Generator/Service/DataCertificate/IDataFELS.php

```diff
@@ -0,0 +1,278 @@
+<?php
+namespace SPHERE\Application\Education\Certificate\Generator\Service\DataCertificate;
+
+use SPHERE\Application\Education\Certificate\Generator\Service\Data;
+use SPHERE\Application\Education\Lesson\Division\Division;
+use SPHERE\Application\Platform\Gatekeeper\Authorization\Consumer\Consumer;
+use SPHERE\Application\Platform\Gatekeeper\Authorization\Consumer\Service\Entity\TblConsumer;
+
+class IDataFELS
```

Comment:
Codacy found an issue: [The class IDataFELS has an overall complexity of 64 which is very high. The configured complexity threshold is 50.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)